core/integration/features/verified/closeout-lognormal.feature

```diff
@@ -209,13 +207,104 @@ Feature: Closeout scenarios
+      | auxiliary1 | ETH/DEC20 | sell | 1      | 10    | 0                | TYPE_LIMIT | TIF_GTC | aux-s-1    |
+    When the opening auction period ends for market "ETH/DEC20"
+    Then the market data for the market "ETH/DEC20" should be:
+      | mark price | trading mode            | auction trigger             | horizon | min bound | max bound | target stake | supplied stake | open interest |
+      | 10         | TRADING_MODE_CONTINUOUS | AUCTION_TRIGGER_UNSPECIFIED | 5       | 10        | 10        | 355          | 400            | 1             |
+      | 10         | TRADING_MODE_CONTINUOUS | AUCTION_TRIGGER_UNSPECIFIED | 10      | 10        | 10        | 355          | 400            | 1             |
+
+    Then the order book should have the following volumes for market "ETH/DEC20":
+      | side | price | volume |
+      | sell | 1002  | 1      |
+      | sell | 1000  | 10     |
+      | buy  | 5     | 5      |
+      | buy  | 1     | 400    |
+
+    When the parties place the following orders with ticks:
+      | party      | market id | side | volume | price | resulting trades | type       | tif     |
+      | trader2    | ETH/DEC20 | sell | 1      | 1003  | 0                | TYPE_LIMIT | TIF_GTC |
+
+    Then the order book should have the following volumes for market "ETH/DEC20":
+      | side | price | volume |
+      | sell | 1003  | 1      |
+      | sell | 1002  | 1      |
+      | sell | 1000  | 10     |
+      | buy  | 5     | 5      |
+      | buy  | 1     | 400    |
+
+    When the parties place the following orders with ticks:
+      | party      | market id | side | volume | price | resulting trades | type       | tif     |
+      | auxiliary2 | ETH/DEC20 | buy  | 12      | 10    | 0                | TYPE_LIMIT | TIF_GTC |
+      | trader2    | ETH/DEC20 | sell | 12      | 10    | 1                | TYPE_LIMIT | TIF_GTC |
+
+    Then the market data for the market "ETH/DEC20" should be:
+      | mark price | trading mode            | auction trigger             | target stake | supplied stake | open interest |
+      | 10         | TRADING_MODE_CONTINUOUS | AUCTION_TRIGGER_UNSPECIFIED | 4623         | 400            | 13            |
+
+    And the parties should have the following margin levels:
+      | party   | market id | maintenance | search     | initial    | release    |
+      | trader2 | ETH/DEC20 | 1560000427  | 2340000640 | 3120000854 | 4680001281 |
+
+    Then the parties should have the following account balances:
+      | party   | asset | market id | margin | general |  
+      | trader2 | USD   | ETH/DEC20 | 999    | 0       | 
+
+    # trader2's order (price 1003) has been canceled
+    Then the order book should have the following volumes for market "ETH/DEC20":
+      | side | price | volume |
+      | sell | 1003  | 0      |
+      | sell | 1002  | 1      |
+      | sell | 1000  | 10     |
```

Comment:
I think the trader is still distressed, there just isn't enough volume on the book to close the position.. Does that conflict with the purpose of the test or is that okay?